js/analytics.md

```diff
@@ -433,6 +432,81 @@ Analytics.record({
 }, "AmazonPersonalize");
 ```
 
+## Using Amazon Kinesis Firehose
+
+The Amazon Kinesis Firehose analytics provider allows you to send analytics data to an [Amazon Kinesis Firehose](https://aws.amazon.com/kinesis/data-firehose) stream for reliably storing data.
+
+### Installation and Configuration
+
+Register the *AWSKinesisFirehoseProvider* with the Analytics category: 
+
+```javascript
+import { Analytics, AWSKinesisFirehoseProvider } from 'aws-amplify';
```

Comment:
Is there any `Amplify.configure(...)` that would need to happen here? Or at least a `Analytics.configure(...)`?

Reply:
The configure steps are here: https://github.com/aws-amplify/docs/pull/1021/files#diff-fec60df6a127d419dccaa0fd86ebb178R475-R493

It follows the same order as the Kinesis docs